.github/workflows/tests.yml

```diff
@@ -36,6 +36,8 @@ jobs:
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Dependency Tracking Info To Datadog
+      run: scripts/track-dependency-status.sh
```

Comment:
In this context, `pip install` has not yet been run, so this is measuring just whatever packages happen to exist on github actions' default python install. To get it to run in the proper context, try moving it to near https://github.com/dimagi/commcare-hq/blob/a3ea4555770f464392f82d8d75174a10d5dc1762/docker/run.sh#L138.

Reply:
(and you do probably want to guard the same way that one is so that it runs only on one job, not on all jobs)

Reply:
Aah! I see. That's why I was curious why does the python package count is coming out so less. I would add it to a later stage.
I will look into ways of guarding it run once.